extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/README.md

```diff
@@ -44,36 +51,58 @@ _**NOTE:**_ *the version should match the aiSSEMBLE project version.*
+
 This option is intended for use in development environments. By setting `ivyCache.storeCacheOnNode` to true, the chart
 will create a PersistentVolume for you that maps to a local directory on the host machine of the Node. This directory
-is defined by the `eventVolume.volumePathOnNode` property.  This allows the Ivy cache to be reused between deployments.
+is defined by the `eventVolume.volumePathOnNode` property. This allows the Ivy cache to be reused between deployments.
 Because the default location is in the `/tmp` directory, the cache may still be cleared between container restarts (e.g.
 when restarting Rancher Desktop).
 
 ### Considerations
-As noted, the local storage option is intended for development environments.  This is due to the inherent security risk 
-of allowing a pod to access the host's filesystem.  In many environments, [`hostPath` volumes](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)
-are disabled by default for this reason.  Moreover, it's important to note that because hostPath and local volumes are
+
+As noted, the local storage option is intended for development environments. This is due to the inherent security risk
+of allowing a pod to access the host's filesystem. In many
+environments, [`hostPath` volumes](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)
+are disabled by default for this reason. Moreover, it's important to note that because hostPath and local volumes are
 tied directly to a single Node, this functionality may not work as expected in mult-node environments.
 
-It is possible to configure the [Pod Security admission controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
-to allow hostPath volumes, either across the board or for Spark History Server pods specifically.  Alternatively, a [`local`
+It is possible to configure
+the [Pod Security admission controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
+to allow hostPath volumes, either across the board or for Spark History Server pods specifically. Alternatively,
+a [`local`
 volume](https://kubernetes.io/docs/concepts/storage/volumes/#local) can be used which alleviates most of the security
 concerns as well as the single-node limitation. However, redeployments of the Spark Operator will not reuse the same
 volume, meaning the cache will be lost between deployments.
+
+# Migration from aiSSEMBLE v1 Helm Charts
+
+If you are migrating from the v1 version of the hive-metastore-service chart, use the tables below to apply any
+existing customizations from the old chart to the new v2 chart.
+
+## Property Location
+All properties listed below have been moved to the parent chart. If any properties are set to the default value, we 
+recommend removing them from your values file entirely.
+
+**Note**: *all new property locations include the prefix `aissemble-spark-history-chart.`*
+
+## Property Removed
+The following properties no longer exist.
```

Comment:
Should there be a table within these sections with the properties?

Reply:
There were no removed properties but left the header to follow the convention. Can remove if we think its confusing.

Reply:
I think we should keep the sections to remain the same as the other README's but maybe mention that's there's just no properties in this instance. For example in the section above `All properties listed below have been moved to the parent chart` makes it seem like we just forgot to list the properties.

Reply:
I like it! Ill update it to include the section but give context as to why its blank